Simplified/Adobe DRM/AdobeDRMContainer.mm

```diff
@@ -0,0 +1,156 @@
+//
+//  AdobeDRMContainer.mm
+//  SimplyE
+//
+//  Created by Vladimir Fedorov on 13.05.2020.
+//  Copyright © 2020 NYPL Labs. All rights reserved.
+//
+
+#import "AdobeDRMContainer.h"
+
+#include <memory>
+#import <ePub3/nav_table.h>
```

Comment:
Yes, files in ePub3 are from readium-sdk repository, adobe filters use objects declared there

Reply:
so does this mean we will need to continue to have this dependency? 
https://github.com/NYPL-Simplified/readium-sdk

Do we also need to continue to depend on these as well?
https://github.com/readium/readium-shared-js
https://github.com/NYPL-Simplified/Adobe-Content-Filter
https://github.com/NYPL-Simplified/DRM-iOS-Adobe

Reply:
- https://github.com/NYPL-Simplified/readium-sdk - yes, it contains ePub3 library we and Adobe-Content-Filter use
- https://github.com/readium/readium-shared-js - not sure what it is, this code doesn't use it
- https://github.com/NYPL-Simplified/Adobe-Content-Filter - yes, filters for decoding ePubs
- https://github.com/NYPL-Simplified/DRM-iOS-Adobe - I don't use it, but doesn't it perform authorisation with user id and device id?

Reply:
regarding DRM-iOS-Adobe... honestly I'm not sure..... @kyles-ep is that what it does, perform authorisation with user id and device id?

Reply:
Adobe DRM support for Readium 2 doesn't contain any Readium 1 code, but I see RDContainer.h and adept_filter.h are used in other parts of older objective-c code. Removing them completely should probably be a different task, linked with migration to Swift

Reply:
DRM-iOS-Adobe, ADEPT auth function is called in NYPLAccountSignInViewController.. not sure if this one is migrated

Reply:
RDContainer.h is probably ok to leave it, it's used in some of client-side R1 reader, so it will go away once we remove R1. I have a task for later to do that.

Regarding adept_filter.h and DRM-iOS-Adobe I think they are used in NYPLAccountSignInViewController and NYPLSettingsAccountDetailViewController to perform device authorization upon sign in. Don't quote me on that though. But in any case, my question is, do we need to update those dependencies as well? It might be outside the scope of this PR, but still needed, and in that case we could file a separate ticket. I just don't know enough about the DRM system, so I'd appreciate your guidance on this.

Reply:
I see, no, we don't need to update these dependences. Adobe DRM for SimplyE use DRM Connector only
- https://github.com/NYPL-Simplified/readium-sdk - not in use in Adobe DRM for SimplyE anymore
- https://github.com/readium/readium-shared-js - not sure what it is, this code doesn't use it
- https://github.com/NYPL-Simplified/Adobe-Content-Filter - not in use in Adobe DRM for SimplyE
- https://github.com/NYPL-Simplified/DRM-iOS-Adobe - not in use in Adobe DRM for SimplyE

Reply:
sounds good -- I'll just leave this open for future reference.